include/otbObiaLSBaatzSegmentationScheduler.txx

```diff
@@ -511,11 +574,15 @@ LSBaatzSegmentationScheduler<TInputImage>
 
     } // end for(uint32_t ty = 0; ty < this->m_NumberOfTilesY; ty++)
 
-    mpiConfig->barrier();
+
+    // mpiConfig->barrier();
+
 
```

Comment:
Since mpi_win_create (L583) seems to be thread safe, we don't need lock and this could be removed safely.